tests/Base.nut

```diff
@@ -0,0 +1,81 @@
+// MIT License
```

Comment:
Now it works, but the base cannot be converted to a class inherited from the ImpTestCase, since the file is connected to both agent and device sides at the same time.